DirectProgramming/DPC++/StructuredGrids/1d_HeatTransfer/src/1d_HeatTransfer.cpp

```diff
@@ -47,11 +48,13 @@ using namespace std;
 constexpr float dt = 0.002f;
 constexpr float dx = 0.01f;
 constexpr float k = 0.025f;
-constexpr float temp = 100.0f;  // Initial temperature.
+constexpr float initial_temperature = 100.0f;  // Initial temperature.
 
-//************************************
-// Function description: display input parameters used for this sample.
-//************************************
+int failures = 0;
+
+//
+// Display input parameters used for this sample.
+//
 void Usage(string programName) {
```

Comment:
Might be better at const string &programName

Reply:
I changed it